src/main/scala/org/bitcoins/core/serializers/transaction/RawTransactionOutputParser.scala

```diff
@@ -17,36 +17,24 @@ import scala.annotation.tailrec
  */
 trait RawTransactionOutputParser extends RawBitcoinSerializer[Seq[TransactionOutput]] with ScriptParser with BitcoinSLogger {
 
+  /** Reads a sequence of outputs, expects first element in the byte array to be a [[org.bitcoins.core.protocol.CompactSizeUInt]]
+    * indicating how many outputs we need to read
+    */
   override def read(bytes : List[Byte]) : Seq[TransactionOutput] = {
+
+    //TODO: Possible bug here, this needs to be parsed as a CompactSizeUInt I think
     val numOutputs = bytes.head.toInt
```

Comment:
We parse this as a comapct size uint inside of `RawTransactionInputParser`, so change this here